documentation/source/users/arkane/input.rst

```diff
@@ -421,6 +421,44 @@ for specifying the same ``rotors`` entry are commented out)::
 
 Note that the atom labels identified within the rotor section should correspond to the indicated geometry.
 
+Multidimensional Hindered Rotors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For multidimensional hindered rotors, ``geometry``, ``frequencies``need to be specified in the same way described in detail in the last section. However,
```

Comment:
Add a space between `''frequencies''` and `need`?

Reply:
done